samples/bot-adaptive-card-actions/csharp/AdaptiveCardActions/AppPackage/manifest.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.12",
```

Comment:
Can we use latest version?

Reply:
yes we can

Reply:
Tested using manifest versions 1.14 and 1.15
its working fine. yes we can use